pkgs/applications/window-managers/picom/picom-jonaburg.nix

```diff
@@ -0,0 +1,13 @@
+{ picom, lib, fetchFromGitHub }:
+
+picom.overrideAttrs (oldAttrs: rec {
```

Comment:
Resolving since it turns out this is an older pattern that is just less commonly used, and we're overriding a lot of things and passing all of them through would be mostly counterproductive.